lib/health_checking/health_checks/health_check.rb

```diff
@@ -0,0 +1,51 @@
+module HealthChecks
+  module HealthCheck
+
+    def self.included(base)
+      base.extend ClassMethods
```

Comment:
The sidekiq example seems decent - it's a little bit unconventional in ruby terms as attributes defined in this way act as methods, circumventing rubys internal attr getters/setters.

I'll spike it out and see how it feels.